.github/workflows/build.yml

```diff
@@ -11,11 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        idf_ver: ["release-v4.1", "release-v4.2", "release-v4.3"]
+        idf_ver: ["release-v4.3"]
         idf_target: ["esp32", "esp32s2"]
```

Comment:
we could also add S3 to the list of targets

Reply:
Already there around line 35.